readme.adoc

```diff
@@ -621,7 +621,7 @@ Following example enables use of Maven Central and add a custom `acme` repositor
 
 [source,java]
 ----
-//REPOS mavenCentral,acme=https://maven.acme.local/maven
+//REPOS mavencentral,acme=https://maven.acme.local/maven
```

Comment:
the lookup should be case insensitive ?

Reply:
is it not working for you?

Reply:
Works, but it's inconsistency with the table just above the this example where `mavencentral` is listed